tests/countries/test_eritrea.py

```diff
@@ -0,0 +1,149 @@
+        cls.government_holidays = Eritrea(categories=GOVERNMENT, years=years)
+        cls.no_estimated_holidays = Eritrea(years=years, islamic_show_estimated=False)
+
+    def test_country_aliases(self):
+        self.assertAliases(Eritrea, ER, ERI)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Eritrea(years=1993))
+
+    def test_new_years_day(self):
+        self.assertHolidayName("New Year's Day", (f"{year}-01-01" for year in range(1994, 2050)))
+
+    def test_orthodox_christmas(self):
+        self.assertHolidayName(
+            "Orthodox Christmas", (f"{year}-01-07" for year in range(1994, 2050))
+        )
+
+    def test_epiphany(self):
+        self.assertHolidayName(
+            "Epiphany",
+            (f"{year}-01-19" for year in range(1994, 2050) if year % 4 != 0),
+            (f"{year}-01-20" for year in range(1994, 2050) if year % 4 == 0),
+        )
+
+    def test_orthodox_easter(self):
+        name = "Orthodox Easter"
+        dt = (
+            "2020-04-19",
+            "2021-05-02",
+            "2022-04-24",
+            "2023-04-16",
+            "2024-05-05",
+            "2025-04-20",
+        )
+        self.assertHolidayName(name, dt)
+        self.assertHolidayName(name, self.no_estimated_holidays, range(1994, 2050))
+
+    def test_ethiopian_new_year(self):
+        self.assertHolidayName(
+            "Ethiopian New Year",
+            (f"{year}-09-11" for year in range(1994, 2050) if year % 4 != 3),
+            (f"{year}-09-12" for year in range(1994, 2050) if year % 4 == 3),
+        )
+
+    def test_finding_of_the_true_cross(self):
+        self.assertHolidayName(
+            "Finding of the True Cross",
+            (f"{year}-09-27" for year in range(1994, 2050) if year % 4 != 3),
+            (f"{year}-09-28" for year in range(1994, 2050) if year % 4 == 3),
+        )
+
+    def test_fenkil_day(self):
+        name = "Fenkil Day"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name, self.government_holidays, (f"{year}-02-10" for year in range(1994, 2050))
+        )
+
+    def test_womens_day(self):
+        self.assertHolidayName("Women's Day", (f"{year}-03-08" for year in range(1994, 2050)))
+
+    def test_independence_day(self):
+        self.assertHolidayName("Independence Day", (f"{year}-05-24" for year in range(1994, 2050)))
+
+    def test_martyrs_day(self):
+        self.assertHolidayName("Martyrs' Day", (f"{year}-06-20" for year in range(1994, 2050)))
+
+    def test_revolution_day(self):
+        self.assertHolidayName("Revolution Day", (f"{year}-09-01" for year in range(1994, 2050)))
+
+    def test_eid_al_fitr(self):
+        name = "Eid al-Fitr"
+        dt = (
+            "2020-05-24",
+            "2021-05-13",
+            "2022-05-02",
+            "2023-04-21",
+            "2024-04-10",
+            "2025-03-30",
+        )
+        self.assertHolidayName(f"{name} (estimated)", dt)
```

Comment:
```suggestion
        self.assertHolidayName(name, self.no_estimated_holidays, range(1994, 2050))
```
And the same for rest of Islamic holidays.